src/Community/Contribute/index.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import CommunityButton from '../Button'
+import CommunityBlock from '../Block'
+import CommunitySection from '../Section'
+
+import data from '../data'
+
+import { Item, Items, Wrapper } from '../styles'
+
+const { description, title } = data.section.contribute
+
+export default function CommunityContribute({ theme }) {
+  return (
+    <Wrapper>
+      <CommunitySection
+        anchor="contribute"
+        background="/static/img/community/contribute_bg.jpg"
+        color={theme.color}
+        description={description}
+        icon="/static/img/community/contribute.svg"
+        title={title}
+      >
+        <Items>
+          <Item>
+            <CommunityBlock
+              title="Make a PR"
+              action={
+                <CommunityButton theme={theme} href="/">
+                  Open Chat
+                </CommunityButton>
+              }
+            >
+              Become DVC contributor and let us build something great together.
+            </CommunityBlock>
+          </Item>
+          <Item>
+            <CommunityBlock
+              title="Write a blogpost"
+              action={
+                <CommunityButton theme={theme} href="/">
+                  Learn More
+                </CommunityButton>
+              }
+            >
+              Have something cool on your mind? Suggest a text and we&apos;ll
+              publish it in our blog.
+            </CommunityBlock>
+          </Item>
+          <Item>
+            <CommunityBlock
+              title="Give a talk"
+              action={
+                <CommunityButton theme={theme} href="/">
+                  Learn More
+                </CommunityButton>
+              }
+            >
+              We support speakers all over the world and help with preparation,
+              speaker training and expenses.
+            </CommunityBlock>
+          </Item>
+          <Item>
+            <CommunityBlock
+              title="Be an Ambassador"
+              action={
+                <CommunityButton theme={theme} href="/">
+                  Apply
+                </CommunityButton>
+              }
+            >
+              Get perks and benefits for significant contributions, creating
+              content or hosting meetups.
```

Comment:
Get perks and benefits for contributing to the code base, writing blogs, hosting meetups and more.

Reply:
Not sure about this "and more" ending here. Maybe "or other community building activities" @andronovhopf?

Reply:
@jorgeorpinel I think "or other community building activities" is too wordy. If we don't like "and more", we could cut it entirely so it reads `Get perks and benefits for contributing to the code base, writing blogs, and hosting meetups.`

Reply:
How about "writing blogs, hosting meetups, etc." ? 🙂

Reply:
Personally, I prefer "and more" to "etc." in blurbs- but that's my stylistic preference. Either way, it's readable and makes sense. No problem if you go ahead with that!

Reply:
Yeah I don't love "etc." either. I guess we can just end the sentence there like you mentioned. "and more" sounds to me like something we're offering, as if this was a list of perks and not of what we wish our ambassadors to help with.

Ok, updated the suggestion above!